VitaminJProcessing.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jun  4 02:05:08 2024
+
+@author: Anupama Rajendra
+"""
+import csv
+import numpy as np
+import pandas as pd
+
+#The purpose of this code is to take the flux vs. energy data derived from OpenMC and
+# restructure it into a form appropriate for ALARA.
+
+#Loading the csv that contains the energy bounds of the Vitamin J structure 
+Vit_J = 'VitaminJEnergyGroupStructure.csv'
+Vit_J_Groups = pd.read_csv(Vit_J, header=1)
```

Comment:
Since this is pretty constant, we probably don't need the intermediate variable
```suggestion
Vit_J_Groups = pd.read_csv('VitaminJEnergyGroupStructure.csv', header=1)
```